commands/test.js

```diff
@@ -109,10 +107,7 @@ function test(projects, options, config) {
     });
 
     const merged = `<?xml version="1.0"?>\n<testsuites>\n${junitTestSuites.join('\n')}</testsuites>\n`;
-
-    exitCode = exec(`mkdir -p ${path.dirname(options.junitOutput)}`);
-    if (exitCode) throw new Error('failed to make junit output dir');
-
+    exitCode = exitCode || exec(`mkdir -p ${path.dirname(options.junitOutput)}`);
```

Comment:
If `exitCode` is 1 (a test failed earlier) then we will never execute the `mkdir`. That's no good. Probably better to write:

```
if (!exec(`mkdir -p ${path.dirname(options.junitOutput)}`)) exitCode = 1;